.bazelrc

```diff
@@ -1,6 +1,3 @@
 import %workspace%/envoy/.bazelrc
 
-build --copt="-DENVOY_SSL_VERSION=\"OpenSSL\""
-build --cxxopt=-Wno-error=old-style-cast
-build --define=boringssl=disabled
```

Comment:
Why did you decide to remove this line and `ENVOY_SSL_VERSION`?

Reply:
@rojkov 

Thanks for the input here. It slipped my attention, let's keep it.

Reply:
@rojkov 

Fixed. Please take a look.

Reply:
I'd preserve `build --define=boringssl=disabled` to explicitly disable all the extensions that depend on BorinSSL.

Reply:
@rojkov 
> I'd preserve build --define=boringssl=disabled to explicitly disable all the extensions that depend on BorinSSL.

Let's keep it then. Done.